echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/AbstractEventNotificationAgent.groovy

```diff
@@ -47,68 +53,71 @@ abstract class AbstractEventNotificationAgent implements EchoEventListener {
 
   @Override
   void processEvent(Event event) {
+    if (log.isDebugEnabled()) {
```

Comment:
Shouldn't the logger only log if debug is enabled anyway?

Reply:
Yes, but the overhead of doing the serialization is avoided with this check because the args are evaluated if you just do `log.debug(arg1, arg2, ... )`